src/config.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  gaTrackingId: process.env.GA_TRACKING_ID || "UA-116475051-1",
```

Comment:
CRA has a special way of dealing with [environment variables](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables). Looks like you should prefix the variable with `REACT_APP_` so in this case:`REACT_APP_GA_TRACKING_ID`.

Reply:
Because you don't like magic (🎉), here's the where it's handled in the ejected branch: https://github.com/streetlives/streetlives-web/blob/ejected_app/config/env.js#L61.

Reply:
Great point; was not aware of it. Thanks!

Reply:
(Also, sorry: no idea how, but I didn't notice the comments until just now. Thank you for reviewing.)